react-components/src/desktop/messages.js

```diff
@@ -30,6 +30,11 @@ var intlData = {
         emailSupportTooltip: "support@cyverse.org",
         supportChatLink: "Chat With Support",
         supportChatTooltip: "Chat may be disabled by your browser's settings",
+        collaboratorsTooltip: "Manage your private set of collaborators",
+        teamsTooltip:
+            "Manage public or private groups of collaborators. Collaborators in a group can all share resources within the group",
+        communitiesTooltip:
+            "Manage public collections of apps for displaying in the Apps window",
```

Comment:
```suggestion
            "Manage public collections of apps for display in Community Categories in the Apps window",
```
This might make it more clear what Communities are for currently, though I'm sure we'll have to change this text if Communities are expanded for other things in the future 🤔

Reply:
Agreed - right now we have wording in the help buttons for Collaborator and Team windows that indicates those groups only work with apps and tools right now as well.